packages/wrangler/src/index.tsx

```diff
@@ -187,32 +187,52 @@ export async function main(argv: string[]): Promise<void> {
         type: "string",
       });
     },
-    async () => {
+    async (args) => {
+      if ("type" in args) {
+        let message =
+          "The --type option is no longer supported.\n" +
+          "The difference between 'javascript' and 'rust' is automatically inferred.";
+        if (args.type === "webpack") {
+          message +=
+            "\nIf you wish to use webpack then you will need to create a custom build";
```

Comment:
When we write docs, we should add a link here to the doc showing how.
```suggestion
            "\nIf you wish to use webpack then you will need to create a custom build.";
```

Reply:
👍